yarn.lock

```diff
@@ -6737,14 +6737,6 @@ shiki@^0.14.7:
     vscode-oniguruma "^1.7.0"
     vscode-textmate "^8.0.0"
 
-shx@0.3.4:
```

Comment:
The older update to dev-scripts caused it to remove 'shx' on the next yarn install.